packages/field-label/README.md

```diff
@@ -1,32 +1,58 @@
-## Description
+## Overview
 
-An `<sp-field-label>` provides accessible labelling for form elements. Use the `for` attribute to outline the `id` of an element in the same DOM tree to which it should associate itself.
+An `<sp-field-label>` provides context to information that a user needs to input. Field labels are commonly used in forms to give users clear guidance about what information is required.
 
 ### Usage
 
 [![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/field-label?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/field-label)
 [![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/field-label?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/field-label)
-[![Try it on webcomponents.dev](https://img.shields.io/badge/Try%20it%20on-webcomponents.dev-green?style=for-the-badge)](https://webcomponents.dev/edit/collection/fO75441E1Q5ZlI0e9pgq/fDq5bpzNQJzUUyEq6ENR/src/index.ts)
 
 ```
 yarn add @spectrum-web-components/field-label
 ```
 
 Import the side effectful registration of `<sp-field-label>` via:
 
-```
+```ts
 import '@spectrum-web-components/field-label/sp-field-label.js';
 ```
 
 When looking to leverage the `FieldLabel` base class as a type and/or for extension purposes, do so via:
 
-```
+```ts
 import { FieldLabel } from '@spectrum-web-components/field-label';
 ```
 
-## Sizes
+### Anatomy
+
+```html
+<sp-field-label for="email">Email address</sp-field-label>
+<sp-textfield id="email" placeholder="user@adobe.com"></sp-textfield>
+```
+
+Field labels can be associated with form elements by using the `for` attribute, which should reference the `id` of the related input element.
 
-<sp-tabs selected="m" auto label="Size Attribute Options">
+```html demo
+<sp-field-label for="email-demo">Email address</sp-field-label>
+<sp-textfield id="email-demo" placeholder="user@adobe.com"></sp-textfield>
+```
+
+Field labels can also be used to label a group of related inputs:
+
+```html demo
+<sp-field-label for="account-type">Account type</sp-field-label>
+<sp-radio-group id="account-type">
+    <sp-radio value="admin">Admin</sp-radio>
+    <sp-radio value="member" checked>Member</sp-radio>
+    <sp-radio value="guest">Guest</sp-radio>
+</sp-radio-group>
+```
+
+### Options
+
+#### Sizes
+
+<sp-tabs selected="m" auto label="Size attribute options">
```

Comment:
Can we have the field sizes match their labels here?

Reply:
i don't think I understand the ask? Im seeing field labels of varying sizes in the correct tab labeled with the same size

Reply:
this matches exactly the same tabs set up as in Button README

Reply:
made some changes, confirm thats what you were wanting @nikkimk